core/data.go

```diff
@@ -40,6 +40,17 @@ type Blob struct {
 	Data          []byte
 }
 
+type BlobAuthHeader struct {
+	// Commitments
+	BlobCommitments `json:"commitments"`
+	// AccountID is the account that is paying for the blob to be stored
+	AccountID AccountID `json:"account_id"`
+	// Nonce
+	Nonce uint32 `json:"nonce"`
+	// AuthenticationData is the signature of the blob header by the account ID
+	AuthenticationData []byte `json:"authentication_data"`
```

Comment:
why don't we call it `Signature`?

Reply:
It's possible this could be more generic in the future... For instance, this could end up being data which gets passed along to the rollup account smart contract, whose semantics the EigenDA protocol is agnostic to. But I'm fine with calling it signature for now.